lib/iex/lib/iex/autocomplete.ex

```diff
@@ -125,6 +128,51 @@ defmodule IEx.Autocomplete do
     end
   end
 
+  defp get_signatures(name, module) when is_atom(module) do
+    with docs when is_list(docs) <- get_docs(module, [:function, :macro], name) do
+      Enum.map(docs, fn {_, _, [signature], _, _} -> signature end)
+    else
+      _ -> :error
+    end
+  end
+
+  defp get_signatures(name, module, fs) when is_atom(module) and is_list(fs) do
+    fs
+    |> Enum.filter(&match?({^name, _arity}, &1))
+    |> Enum.map(fn {name, _} -> {module, name} end)
+    |> Enum.uniq()
+    |> Enum.map(fn {module, name} -> get_signatures(name, module) end)
+  end
+
+  defp expand_help(expr, server) do
+    signatures =
+      case Code.string_to_quoted(expr) do
+        {:ok, {{:., _, [{:__aliases__, _, mod}, fun]}, _, []}} when is_atom(fun) ->
+          with {:ok, mod} <- expand_alias(mod, server) do
+            get_signatures(fun, mod)
+          end
+
+        {:ok, {fun, _, _}} when is_atom(fun) ->
+          imports_from_env(server)
+          |> Enum.map(fn {module, fs} -> get_signatures(fun, module, fs) end)
+          |> List.flatten()
+
+        {:ok, {{:., _, [mod, fun]}, _, []}} when is_atom(mod) and is_atom(fun) ->
+          get_signatures(fun, mod)
+
+        _ ->
+          :error
+      end
+
+    with [_ | _] <- signatures do
+      [head | tail] = Enum.sort(signatures, &(String.length(&1) <= String.length(&2)))
+      if tail !== [], do: IO.write("\n" <> (tail |> Enum.reverse() |> Enum.join("\n")))
```

Comment:
We generally use `!=` in Elixir, unless `!==` is necessary.

```suggestion
      if tail != [], do: IO.write("\n" <> (tail |> Enum.reverse() |> Enum.join("\n")))
```